index.html

```diff
@@ -9889,7 +9913,7 @@ <h4 id="Beast Shape - Medium Animal">Beast Shape - Medium Animal</h4>
                </td>
                <td>Knock to the Floor
                </td>
-               <td>Knock and enemy within 5ft <a class="condition" data-tooltip-container>prone</a>. 2 Fort resist
+               <td>Knock an enemy within 5ft <a class="condition" data-tooltip-container>prone</a>. 2 Fort resist
```

Comment:
Again, we can't have an infinite use source of a condition, or 2 resistance drain.